src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/ContinueSetupForSingleApprenticeship.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@using SFA.DAS.EmployerAccounts.Web.Extensions
+@using SFA.DAS.EmployerAccounts.Web.Helpers
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
+
+<section class="dashboard-section">
+    <h2 class="section-heading heading-large" id="call-to-action-continue-setting-up-an-apprenticeship">
+        Continue setting up an apprenticeship
+    </h2>
+
+    <table class="responsive">
+        <tr>
+            <th scope="row" class="tw-35">
+                Course
+            </th>
+            <td class="tw-65" id="course-name">
+                @(Model?.CallToActionViewModel?.CourseName)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Training Provider
+            </th>
+            <td id="training-provider">
+                @(Model?.CallToActionViewModel?.ProviderName)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row" class="govuk-tag--inactive">
```

Comment:
class should be applied to the <td> not the <th>

Reply:
done